auth_style/templates/auth_style/base.html

```diff
@@ -20,18 +20,7 @@
     {% block body %}
       <div class="flex flex-col sm:flex-row sm:navbar">
         <span class="text-center sm:text-left sm:grow">
-          {% setvar site_name %}
-            {% if request.site %}
-              {{ request.site.name }}
-            {% elif site %}
-              {{ site.name }}
-            {% endif %}
-          {% endsetvar %}
-          {% if site_name %}
-            <span class="text-2xl font-bold">
-              {{ site_name }}
-            </span>
-          {% endif %}
+          {% include "auth_style/components/site_name.html" %}
```

Comment:
No need to put this under `components/`. That's just a thing we had to do for the templates that override things in Django Allauth.

This should still be under `auth_style/`, since it's a template name that we're inventing, not something from Allauth being overridden.